src/models/stocks.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+import { users } from "./user";
+import type { IStock } from "@interfaces";
+import { mongoose } from "@services";
+import { STOCK_API_URL } from "@src/types/values";
+
+class Stocks extends mongoose.Repository<IStock> {
+	private static readonly stockApiClient = axios.create({
+		baseURL: STOCK_API_URL
+	});
+
+	constructor() {
+		super("Stock", {
+			server_id: {
+				type: String,
+				index: true,
+				required: true
+			},
+			user: {
```

Comment:
lets call this `user_id` and just make it a string, I'm trying to keep the number of refs minimal for now